core/src/main/java/com/yahoo/oak/BlocksPool.java

```diff
@@ -97,6 +97,16 @@ static void setBlockSize(int blockSize) {
         }
     }
 
+    static void clear() {
```

Comment:
Before, I've noticed that the memory usage during testing is too high. I've fixed some tests to correctly close Oak. But I also wanted to clear all the used blocks to release the memory.
This function is used for testing to release all the blocks but still leave the pool operational.
I think that we should expose this API to the users due to Issue #192 .

Reply:
My understanding of `clear()` (e.g. vs `close()`) is that it makes the data structure "clear", but keeps the structure so it can be "filled" again. Thus the entire memory is not released. Thus issue #192 requires another API to resolve the user request there. Let's discuss.